--- dwarftherapist/src/dfinstancelinux.cpp	2010-06-28 07:51:20.372206589 +0200
+++ dwarftherapist-build/src/dfinstancelinux.cpp	2010-06-28 08:08:37.295026710 +0200
@@ -135,13 +135,13 @@
     return ret_val;
 }
 
-uint DFInstanceLinux::write_string(const uint &addr, const QString &str) {
+int DFInstanceLinux::write_string(const VIRTADDR &addr, const QString &str) {
     Q_UNUSED(addr);
     Q_UNUSED(str);
     return 0;
 }
 
-uint DFInstanceLinux::write_int(const uint &addr, const int &val) {
+int DFInstanceLinux::write_int(const VIRTADDR &addr, const int &val) {
     return write_raw(addr, sizeof(int), (void*)&val);
 }
 
@@ -275,7 +275,7 @@
 }
 */
 
-uint DFInstanceLinux::write_raw(const uint &addr, const uint &bytes, void *buffer) {
+int DFInstanceLinux::write_raw(const VIRTADDR &addr, const int &bytes, void *buffer) {
     // try to attach, will be ignored if we're already attached
     attach();
 
--- dwarftherapist/inc/dfinstancelinux.h	2010-06-28 07:51:20.348873106 +0200
+++ dwarftherapist-build/inc/dfinstancelinux.h	2010-06-28 08:12:08.001664993 +0200
@@ -41,7 +41,7 @@
     QString read_string(const VIRTADDR &addr);
 
     // Writing
-    int write_raw(const VIRTADDR &addr, const uint &bytes, void *buffer);
+    int write_raw(const VIRTADDR &addr, const int &bytes, void *buffer);
     int write_string(const VIRTADDR &addr, const QString &str);
     int write_int(const VIRTADDR &addr, const int &val);
 
